tools/taco.cpp

```diff
@@ -734,6 +735,46 @@ int main(int argc, char* argv[]) {
     evaluate = lower(stmt, "evaluate", true, true);
   }
 
+  string packComment = 
+    "/*\n"
+    " * The `pack` functions convert coordinate and value arrays in COO format to\n"
+    " * the specified input format.\n"
```

Comment:
Should note that input nonzeros need to be sorted lexicographically by their coordinates.

Reply:
Added this to comment.